packages/create-toolpad-app/src/generateProject.ts

```diff
@@ -279,9 +194,9 @@ export default function generateProject(
       'react-dom': '^18',
       next: '^14',
       '@toolpad/core': 'latest',
-      '@mui/material': '^5',
+      '@mui/material': '^6',
```

Comment:
@apedroferreira I believe this will broken, this version doesn't exist on npm.

Reply:
Ok thanks for finding it, I'm working on bug fixes so will make sure the CLI and generated project work along with them.

Reply:
I changed these ones to `'next'` in https://github.com/mui/mui-toolpad/pull/3807

Reply:
Oh, didn't know you were releasing or might have checked more carefully. Thanks!